src/code42cli/cmds/securitydata_output_formats.py

```diff
@@ -0,0 +1,100 @@
+from datetime import datetime
+
+from c42eventextractor.logging.formatters import CEF_TEMPLATE
+from c42eventextractor.logging.formatters import CEF_TIMESTAMP_FIELDS
+from c42eventextractor.maps import CEF_CUSTOM_FIELD_NAME_MAP
+from c42eventextractor.maps import FILE_EVENT_TO_SIGNATURE_ID_MAP
+from c42eventextractor.maps import JSON_TO_CEF_MAP
+
+import code42cli.cmds.search.enums as enum
+from code42cli.cmds.search.output_formats import extraction_output_format
+from code42cli.output_formats import CEF_DEFAULT_PRODUCT_NAME
+from code42cli.output_formats import CEF_DEFAULT_SEVERITY_LEVEL
+
+
+def file_events_output_format(_, __, value):
```

Comment:
are there any tests around `to_cef`? I know this is largely just a copy/paste over from the extractor -- the tests from there could also be moved over, or we could make our own here. They probably don't have to be exhaustive as what we have in the extractor, but something that at least does a sanity check that proves that converting to CEF doesn't bomb completely is probably a good idea.

Reply:
Yes, there are. I copied all the tests over.

Reply:
`cmds/test_securitydata_output_formats.py`

Reply:
ah thanks, i missed it because the diff was large so github didn't show it.

Reply:
That is because we are so good at writing tons of tests haha